src/app/docs/tabs/tabs-docs.component.html

```diff
@@ -4,10 +4,373 @@
   pageTitle="Tabs"
 >
   <sky-docs-demo-page-summary>
-    Sample description.
+    Tabs are containers for dividing large amounts of content on a page.
   </sky-docs-demo-page-summary>
 
+  <sky-docs-demo>
+    <sky-docs-demo-control-panel
+      (selectionChange)="onDemoSelectionChange($event)"
+    >
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-checkbox
+          label="Include counts"
+          propertyName="includeCounts"
+          [checked]="false"
+        >
+        </sky-docs-demo-control-panel-checkbox>
+      </sky-docs-demo-control-panel-section>
+
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-checkbox
+          label="Include add and close"
+          propertyName="includeAddAndClose"
+          [checked]="false"
+        >
+        </sky-docs-demo-control-panel-checkbox>
```

Comment:
@Blackbaud-AlexKingman, my understanding is that you are addressing the behavior of this option so that it resets the demo as well as the other settings. Is that correct? And if so, can I go ahead and resolve this comment?

Reply:
@blackbaud-johnly that is correct! This will require a tweak to docs-tools in order to get it to work as you describe. I've filed this to track that work: https://github.com/blackbaud/skyux-docs-tools/issues/20